plotly/tests/test_core/test_graph_objs/test_properties_validated.py

```diff
@@ -102,6 +102,16 @@ def test_present_colorscale(self):
         # Presented as tuple of tuples
         self.assertEqual(self.scatter.marker.colorscale,
                          ((0, 'red'), (1, 'green')))
+    
+    def test_present_colorscale_str(self):
+        self.assertIsNone(self.scatter.marker.colorscale)
+
+        # Assign string
+        self.scatter.marker.colorscale = "Viridis"
+
+        # Presented as a string
+        self.assertEqual(self.scatter.marker.colorscale,
+                         "Viridis")
```

Comment:
In addition to this test, we should update the `test_acceptance_named` validator test in `_plotly_utils/tests/validators/test_colorscale_validator.py`

It should work a bit more like the `test_acceptance_array` test in that same file (the second one, it's a mistake that there are two with the same name). Where the coerced value is held onto and then run through the validators `present` method.

If it's not obvious how this should work don't spend much time on it, I need to do some cleanup in there soon anyway.

Reply:
I've gone ahead and updated that test. Would it be worthwhile to set the `named_colorscale` fixtures possible values to be set either by `ColorScaleValidator.named_colorscales` or some schema to be sure it's up to date?

Also, should Cividis be on those lists?

Reply:
I think I'd like to keep the explicit list of named color scales in the tests. It is duplicated with the list in `ColorScaleValidator.named_colorscales`, but if we ever try to pull the validator's named colorscales list out of the schema I'd like to have this explicit list in the tests

Yes, good call on Cividis. Could you add that to both the test fixture and the `named_colorscales` list? Thanks!

Reply:
Added. I think that's everything, right?